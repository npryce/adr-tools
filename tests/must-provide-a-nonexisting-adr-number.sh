adr new The First Decision
adr new The Second Decision
adr new The Third Decision
if adr new -n 2 The Fourth Decision
then
    echo ERROR: should have failed
fi
