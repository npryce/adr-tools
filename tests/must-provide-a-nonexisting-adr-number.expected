adr new The First Decision
doc/adr/0001-the-first-decision.md
adr new The Second Decision
doc/adr/0002-the-second-decision.md
adr new The Third Decision
doc/adr/0003-the-third-decision.md
if adr new -n 2 The Fourth Decision
then
    echo ERROR: should have failed
fi
ERROR: ADR number 2 already exists
